SPECS/qtbase/qtbase-moc-macros.patch

```diff
@@ -0,0 +1,28 @@
+From 623a7c414276266e8ccbae7608398489ee96c1cd Mon Sep 17 00:00:00 2001
+From: Sam Meluch <sammeluch@microsoft.com>
+Date: Mon, 29 Jan 2024 13:49:01 -0800
+Subject: [PATCH] moc macros patch
```

Comment:
might not need this (I believe is was made to support multilib according to the comment in the spec file).  See my comment for that,.

Reply:
removed!